server/app.js

```diff
@@ -35,5 +37,8 @@ app.use(function(err, req, res, next) {
   res.status(err.status || 500);
   res.json({ error: err });
 });
-
+const port = process.env.PORT || 3001;
+app.listen(port, () => {
+  console.log("Server is listening on post 3001.");
+});
```

Comment:
The server is being run in `bin/www` so adding a listener here is not necessary. you can use `npm run dev` to start the server

Reply:
Removed the app.listen listener from server/app.js